MultiModalCLF/MoodBoardMultiModalClassification.ipynb

```diff
@@ -0,0 +1,3617 @@
+            "  warnings.warn(\n"
+          ]
+        },
+        {
+          "output_type": "display_data",
+          "data": {
+            "text/plain": [
+              "tokenizer_config.json:   0%|          | 0.00/314 [00:00<?, ?B/s]"
+            ],
+            "application/vnd.jupyter.widget-view+json": {
+              "version_major": 2,
+              "version_minor": 0,
+              "model_id": "d1b130818cf941e3b949bffff5ed2a6a"
+            }
+          },
+          "metadata": {}
+        },
+        {
+          "output_type": "display_data",
+          "data": {
+            "text/plain": [
+              "vocab.txt:   0%|          | 0.00/232k [00:00<?, ?B/s]"
+            ],
+            "application/vnd.jupyter.widget-view+json": {
+              "version_major": 2,
+              "version_minor": 0,
+              "model_id": "7bf90b470124458c89d008b71154e6da"
+            }
+          },
+          "metadata": {}
+        },
+        {
+          "output_type": "display_data",
+          "data": {
+            "text/plain": [
+              "tokenizer.json:   0%|          | 0.00/711k [00:00<?, ?B/s]"
+            ],
+            "application/vnd.jupyter.widget-view+json": {
+              "version_major": 2,
+              "version_minor": 0,
+              "model_id": "b2dc9123594548ca9a9c54201be41ccc"
+            }
+          },
+          "metadata": {}
+        },
+        {
+          "output_type": "display_data",
+          "data": {
+            "text/plain": [
+              "special_tokens_map.json:   0%|          | 0.00/125 [00:00<?, ?B/s]"
+            ],
+            "application/vnd.jupyter.widget-view+json": {
+              "version_major": 2,
+              "version_minor": 0,
+              "model_id": "3e3afeed03384254be9d5e8d32c7cd73"
+            }
+          },
+          "metadata": {}
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "data = pd.read_csv('trimmed_valid_images.csv')\n",
+        "train_set, test_set = train_test_split(data, test_size=0.2, random_state=42)  # 20% for testing\n",
+        "train_set.to_csv('train_data.csv', index=False)\n",
+        "test_set.to_csv('test_data.csv', index=False)"
+      ],
+      "metadata": {
+        "id": "R0jr2NkEQ_IG"
+      },
+      "execution_count": 10,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Dataset objects to feed into trainer\n",
+        "train_dataset = MoodBoardDataset(csv_file='train_data.csv')\n",
+        "test_dataset = MoodBoardDataset(csv_file='test_data.csv')"
+      ],
+      "metadata": {
+        "id": "iQIPhbjlWY67"
+      },
+      "execution_count": 11,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "minidata = pd.read_csv('test_data.csv')\n",
+        "minitrain, minitests = train_test_split(minidata, test_size=0.3, random_state=42)\n",
```

Comment:
This can likely be deleted but can be a test for the user if you would like to run a random subset of the dataset, as the original dataset is quite large and will take several hours to run